pkg/controller/controller_instance.go

```diff
@@ -290,12 +290,12 @@ func (c *controller) reconcileServiceInstanceDelete(instance *v1alpha1.ServiceIn
 		}
 	}
 
-	glog.V(4).Infof("Deprovisioning ServiceInstance %v/%v of ClusterServiceClass %v at ClusterServiceBroker %v", instance.Namespace, instance.Name, serviceClass.Name, brokerName)
+	glog.V(4).Infof(`ServiceInstance "%v/%v": Deprovisioning ClusterServiceClass %v at ClusterServiceBroker %v`, instance.Namespace, instance.Name, serviceClass.Name, brokerName)
```

Comment:
Can you disambiguate the k8s name and human-readable name of the service class?

Reply:
We should clarify here that we're not deprovisioning the ClusterServiceClass, we're deprovisioning this instance OF that class.

Reply:
I think the message should just be "Deprovisioning"

Reply:
Sounds good to me.